src/Core/Addon/Module/ModuleInterface.php

```diff
@@ -30,6 +30,12 @@
 
 interface ModuleInterface extends AddonInterface
 {
+    public function getInstance();
+
+    public function hasValidInstance();
+
+    public function get(string $name);
+
```

Comment:
Please document the BC Break

Reply:
Isn't it possible to typehint the return of methods ?

Reply:
The `Core/Addon` namespace will be completely removed in the 8.0

Reply:
It's currently a BC Break. The removal has not been done.